src/DoctrineMessageRepository/UuidSerializer.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EventSauce\MessageRepository\DoctrineMessageRepository;
+
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+class UuidSerializer
```

Comment:
It felt really weird to have `interface UuidSerializer` for such a simple class. I am happy to modify if desired.